lineage/tests/test_tHMM.py

```diff
@@ -102,3 +102,17 @@ def test_get_leaf_NF(self):
         for ind, NFlin in enumerate(NF):
             self.assertGreaterEqual(NFlin.shape[0], 0)  # at least zero cells in each lineage
             self.assertGreaterEqual(NF3[ind].shape[0], 0)
+            
+    def test_level_of_performance(self):
+        """
+        Really defined states should get an accuracy >95%. 
+        Lineages used should be large and distinct.
+        """
+        X = [LineageTree.init_from_parameters(pi, T, E, (2**11))]
```

Comment:
You need to define `pi`, `T`, and `E`. Either use the ones defined in `self`, that is `self.pi`, `self.T`, and `self.E` or define them inside this funciton.